bindings/translate-time-profile.hh

```diff
@@ -24,7 +24,7 @@ namespace dd {
 v8::Local<v8::Value> TranslateTimeProfile(
     const v8::CpuProfile* profile,
     bool includeLineInfo,
-    ContextsByNode* contextsByNode = nullptr,
+    std::shared_ptr<ContextsByNode> contextsByNode = nullptr,
```

Comment:
nit: you could have kept a raw pointer for `contextsByNode` in `translate-time-profile.cc/hh`

Reply:
Yeah, but figured why not be consistent. If I already have a safe shared pointer around I might as well use it.